### ISSUE TITLE
Cacti Notification Reboot - possibility to ommit notifications (devices,sites) by timerange or by hand

### ISSUE THREAD

**Issue by interduo** [Cacti/plugin_maint#8]
If We need to use it fast as every day action we need three things:

**1. Option in host form:**
![expeted_1](https://user-images.githubusercontent.com/17087236/53742854-e38b8200-3e99-11e9-971e-f97e6d619df7.png)

**2. Option in devices list form in dropdown menu:**
![zrzut ekranu z 2019-03-04 16-23-05](https://user-images.githubusercontent.com/17087236/53743093-5b59ac80-3e9a-11e9-8ed1-3a7f75e2bec0.png)

**3. Option in sites.php to add whole site to maintance by one click somehow (now+1h).**

**4. Option to set maintance mode for devices for specified branch from tree in monitor.php.**

**5. Show devices in maintance mode in monitor.php with other icon (or icon 'screwdriver' from font awesome on the top).**

**6. Enable/Disable maintance mode switch in device.php.**
Something like this.
![maintance](https://user-images.githubusercontent.com/17087236/53820311-cf17ba00-3f6b-11e9-85e8-72055874a975.png)

**Comment by interduo** [comment]
I've updated first post after consulting issue with few persons.

**Comment by interduo** [comment]
Why this plugin is not integrated in Cacti?

**Comment by netniV** [comment]
That is a good question and primarily probably because not everyone uses the functionality of thold and monitor so wouldn't uses this.  It may be worth splitting out each request to be implemented for better clarity.  These seem like reasonable ideas.

**Comment by interduo** [comment]
Using of thlod and monitor isn't obligatory even if it's installed.

**Comment by astrayel** [comment]
I agree with all of this.

**Comment by TheWitness** [comment]
Wow, 6 years ago and though it was written, no pull request?

It's not a ton of work.

**Comment by astrayel** [comment]
I think I will. Stay tuned. 😉

**Comment by astrayel** [comment]
I don't think we can do the #3 and #4 because there is no hook for sites and trees.
